dev/a11y_assessments/lib/use_cases/badge.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'use_cases.dart';
+
+class BadgeUseCase extends UseCase {
+
+  @override
+  String get name => 'Badge';
+
+  @override
+  String get route => '/badge';
+
+  @override
+  Widget build(BuildContext context) => const MainWidget();
+}
+
+class MainWidget extends StatefulWidget {
+  const MainWidget({super.key});
+
+  @override
+  State<MainWidget> createState() => MainWidgetState();
+}
+
+class MainWidgetState extends State<MainWidget> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Not sure what to do with auto focus in this case.

Reply:
Is the badge not focusable? It seems like a component that should be interactive.

Reply:
badge by it self doesn't have a gesture recognizer. Even if it has a gesture detector, we still need some widget to possess focusnode.